test/e2e/examples_test.go

```diff
@@ -65,11 +112,11 @@ func TestSuperTuxKartGameServerReady(t *testing.T) {
 	assert.Equal(t, agonesv1.GameServerStateReady, readyGs.Status.State)
 }
 
-func TestRustGameServerReady(t *testing.T) {
+func TestCppSimpleGameServerReady(t *testing.T) {
```

Comment:
How come one of these has flipped with the other? 🤔 this is interesting?

Reply:
I changed the order😁

Reply:
You can revert this - there's no reason I can think of to change the order, and gives us a cleaner commit history.